examples/blocked_cholesky_gpu.py

```diff
@@ -59,21 +64,19 @@ def ltriang_solve(a, b):
         raise ValueError("Input array shapes are not compatible.")
     if a.shape[0] != a.shape[1]:
         raise ValueError("Array for back substitution is not square.")
-    # For the implementation here, just assume lower triangular.
+    #For the implementation here, just assume lower triangular.
```

Comment:
I think the style guides for Python generally have a space after `#` in comments, so there is no reason to remove all said spaces. ;-) This is not important though. Just a nit.

Reply:
This is correct. The spaces should be there.